app/assets/javascripts/admin/addon/components/dashboard-new-feature-item.hbs

```diff
@@ -1,17 +1,16 @@
 <div class="admin-new-feature-item">
-  <div class="new-feature-emoji">{{this.item.emoji}}</div>
   <div class="new-feature-content">
     <div class="header">
-      {{#if this.item.link}}
-        <a
-          href={{this.item.link}}
-          target="_blank"
-          rel="noopener noreferrer"
-        >{{this.item.title}}</a>
-      {{else}}
-        {{this.item.title}}
+      <h3>{{this.item.title}}</h3>
+      {{#if this.item.emoji}}
+        <div class="new-feature-emoji">{{this.item.emoji}}</div>
```

Comment:
Same comments as the other PR on CSS classes, I think now would be a good time to BEM-ify them.

Also it's probably a good time to convert a component this small to `.gjs` format